docs/DeveloperGuide.adoc

```diff
@@ -800,39 +802,57 @@ Priorities: High (must have) - `* * \*`, Medium (nice to have) - `* \*`, Low (un
 |Priority |As a ... |I want to ... |So that I can...
 |`* * *` |new user |see usage instructions |refer to instructions when I forget how to use the App
 
```

Comment:
Missed out the `So that I can...` part.

Perhaps a description can be `Manage details of students and schedule interviews/job offers.`